src/compiler/gcc.rs

```diff
@@ -83,7 +83,7 @@ pub const ARGS_WITH_VALUE: &'static [&'static str] = &[
     "-iframework", "-imacros", "-imultilib", "-include",
     "-install_name", "-iprefix", "-iquote", "-isysroot",
     "-isystem", "-iwithprefix", "-iwithprefixbefore",
-    "-u",
+    "-u", "-x", "-arch", "--serialize-diagnostics",
```

Comment:
Thanks for the patch! This looks reasonable, but shouldn't these go in clang.rs if they're clang-specific?